src/daemon/mod.rs

```diff
@@ -709,14 +708,168 @@ fn create_password(prompt: &str) -> std::io::Result<String> {
         .interact_on(&term)
 }
 
+pub async fn open_forest_car_union_db(
```

Comment:
I'd try to not bloat this file any further, it's already quite big. Maybe let's have a dedicated one?

Reply:
Moved to `db_util.rs`